docs/concepts/pdp/factdb.mdx

```diff
@@ -0,0 +1,230 @@
+  -H 'Content-Type: application/json' -H 'Accept: application/json' \
+  -H 'Authorization : Bearer {api_key}' \
+  --data-raw '{
+      "settings": {
+            "factdb_enabled":true,
+            "allow_legacy_data_store":true
+      }
+    }'
+```
+
+With this setting enabled, you can run both FactDB PDPs with `PDP_FACTDB_ENABLED=true` and Legacy PDPs in the same environment.
+
+:::info
+We recommend you to migrate all your PDPs in the environment to FactDB.
+Please contact us on [Slack](https://io.permit.io/slack) for more information.
+:::
+
+
+### PDP Offline Mode with FactDB
+
+FactDB can be used in conjunction with the [PDP Offline Mode](/how-to/deploy/deploy-to-production/#pdp-offline-mode) feature.
+When the PDP is in Offline Mode, the FactDB will boot from the locally stored disk volume, preserved from previous runs.
+
+Here is an example of how to start the PDP in Offline Mode with FactDB enabled.
+
+1. Create a volume for the PDP backup
+```bash
+docker volume create pdp-offline-backup
+```
+
+2. Start a new PDP container with the volume attached at `/app/backup`
+```bash
+docker run -it -p 7766:7000 \
+  --name pdp-online \
+  --env PDP_API_KEY=<YOUR_API_KEY> \
+  --env PDP_FACTDB_ENABLED=true \
+  --env PDP_ENABLE_OFFLINE_MODE=true \
+  -v pdp-offline-backup:/app/backup:rw \
+  permitio/pdp-v2:latest
+```
+
+With the backup volume attached, even when the PDP is unable to connect to the Permit API,
+it can still boot up with the FactDB, policies and configurations from the previous run.
+
+```bash
+docker run -it -p 7766:7000 \
+  --name pdp-offline \
+  --network none \
+  --env PDP_API_KEY=<YOUR_API_KEY> \
+  --env PDP_FACTDB_ENABLED=true \
+  --env PDP_ENABLE_OFFLINE_MODE=true \
+  -v pdp-offline-backup:/app/backup:rw \
+  permitio/pdp-v2:latest
+```
+
+## FactDB Architecture
+
+When the PDP boots, FactDB downloads the latest snapshot, and keep it up-to-date from the Permit Cloud and other PDPs in the environment.
+
+Using the stored data, when the PDP receive a query, it queries the FactDB for the relevant data, then passes it to the OPA engine for policy evaluation.
+
+```mermaid
+sequenceDiagram
+    participant cloud as Permit Cloud
+    participant pdp as PDP
+    participant factdb as FactDB
+    participant opa as OPA
+    cloud  ->> pdp: Download Snapshot
+    pdp ->> factdb: Load Snapshot
+    cloud ->> factdb: Send Updates since Snapshot
+    pdp ->> factdb: Query Data
+    pdp ->> opa: Evaluate Policy using the Data
+```
+
+
+## Troubleshooting & Debugging
+
+### Refreshing Data
+Every time the PDP boots, it downloads the latest snapshot from the Permit Cloud and stores it in the `/app/backup/` directory.
+
+By restarting the PDP, you can refresh the FactDB data with the most up-to-date data from the Permit Cloud.
+
+### Extracting Data
+PDP with FactDB stores data in the `/app/backup/` directory. You can inspect the data by mounting the volume to your local machine.
```

Comment:
Should be more like:
To extract the FactsDB store in the PDP, just mount the `app/backup` volume in your local machine and use the data locally.